docs/DeveloperGuide.md

```diff
@@ -196,10 +196,134 @@ H  I     J  K      L  M      N  O
 ```
 
 
+
+
+## Implementation of Logic Gate Commands
```

Comment:
BTW should we add this before the binarytree part? Because I've referenced Gates and all in that section

Reply:
Yes.